tests/foreman/cli/test_activationkey.py

```diff
@@ -1,7 +1,8 @@
 # -*- encoding: utf-8 -*-
+# pylint: disable=too-many-public-methods, too-many-lines
+# pylint: disable=unexpected-keyword-arg
```

Comment:
this is specifically because we pass `cached=True` in cli factory calls and pylint errors saying the argument is not valid.